.gitignore

```diff
@@ -20,3 +20,6 @@ Session.vim
 .netrwhist
 *~
 tags
+VirtualBox-5.0.10-r104061-MultiArch_amd64.msi
```

Comment:
these files should not need to be put into the `.gitignore`, they should simply live in a different dir (like your download directory or something).